### ISSUE TITLE
Durability doesn't go down

### ISSUE THREAD

**Issue by HenryMaxler** [hammertater/treechop#2]
**Describe the bug**
I love the mod, but I was chopping trees with a wood ax and the durability never went down, even when I felled a huge tree.  I don't think it should break from each wood chopped, but not going down at all seems like something is wrong

**Versions**

Version: TreeChop-1.16.4-0.5
Forge: forge-35.1.4

**Comment by hammertater** [comment]
Yikes! You're right, durability is not being reduced. I will fix this as soon as I can. Thanks for reporting this!

I did intend for durability to be reduced with each chop, rather than only when the tree is felled. This way, 1) an axe can break midway through chopping a tree (like in real life!) without losing your progress, 2) if multiple players work together to chop down a tree, they all share the loss in durability, and 3) this approach avoids the exploit in other tree chopping mods that allow you to fell a really big tree with the last charge of a wooden axe (or even worse, only break a fraction of the blocks in the tree, leaving the rest awkwardly floating too high to reach).

**Comment by hammertater** [comment]
This issue should be fixed as of https://github.com/hammertater/treechop/releases/tag/0.6

**Comment by HenryMaxler** [comment]
Hey yeah, it's fixed.  It's basically perfect as far as I'm concerned.  I did notice some weird things like if I chop a tree next to my wall made of different kind of logs it cut those logs as well (and maybe they disappeared 1 time).  If you chop a wall and make a mistake, the stripped log can then be chopped by crouching.  But, then it doesn't drop anything.  But like I said before overall it's kinda like perfect.  Also when I chopped a row of oak trees, going perpendicular to the row, it looked kinda funny.  I'm not sure if that's a new thing that is happening or if I can do it again.   I'll eventually try it again.  Yeah I'd like to be able to continue chopping dead trees and walls without leaves.  You could even do like a wood chopping type competition.  So if it only chops with leaves I'll probably be configuring it off.

**Comment by hammertater** [comment]
> if I chop a tree next to my wall made of different kind of logs it cut those logs as well

I allowed different kinds of logs to break because I've seen some modded trees that can be made of more than one type of log. I'll add a config option to keep different types of logs from breaking. See #11.

> If you chop a wall and make a mistake, the stripped log can then be chopped by crouching. But, then it doesn't drop anything.

The first time you chop a log, it's supposed to drop an item, but there was a bug in 0.8. It's fixed in 0.9. Sorry about that!

> Also when I chopped a row of oak trees, going perpendicular to the row, it looked kinda funny.

Ahhhh, I only made models for logs connected vertically. I'll see if I can add some models for horizontal logs. See #12.

> So if it only chops with leaves I'll probably be configuring it off.

No need to worry, I'll keep leafless trees choppable by default (#7).

I'm gonna close this issue since the durability bug is fixed, but feel free to keep chatting in #11 and #12 or open a new issue if you'd like. Thanks for all the feedback! It really helps!
